controllers/common.go

```diff
@@ -20,11 +20,11 @@ import (
 
 	mcev1 "github.com/stolostron/backplane-operator/api/v1"
 
-	searchv2v1alpha1 "github.com/stolostron/search-v2-operator/api/v1alpha1"
-
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/library-go/pkg/operator/resource/resourcemerge"
 	subv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	searchv2v1alpha1 "github.com/stolostron/search-v2-operator/api/v1alpha1"
+	slice "golang.org/x/exp/slices"
```

Comment:
Golang describes the packages in `exp` as "experimental and unreliable" so I don't think we can rely on them.

Reply:
https://github.com/stolostron/multiclusterhub-operator/blob/main/pkg/utils/utils.go#L444 might be able to replace the use of slice.Contains